scipy/sparse/linalg/tests/test_interface.py

```diff
@@ -448,3 +448,8 @@ def test_transpose_noconjugate():
 
     assert_equal(B.dot(v), Y.dot(v))
     assert_equal(B.T.dot(v), Y.T.dot(v))
+
```

Comment:
Linter complains about a single and not double blank line in tests, however I copied the formatting from surrounding code. Should I change this to two lines?

Reply:
Yes 2 lines are required between standalone functions. The linter is just looking at current changes. This is why it cannot spot old mistakes 😅

Reply:
I understand, but what is the preferred way to go:

- apply the new style and have inconsistent formatting
- leave 1 line
- fix this part of formatting in the whole module in a separate commit

Reply:
For sure any new code should pass the linter/tests. Usually we do not want people to reformat what they don't touch as to not break the git history and blame. If it's just a few things fine it's ok, but if the diff is too large, please don't change it.

Reply:
For now I went with the first option to make tests pass, however happy to revert to either of the other alternatives.